docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>mention those helper functions are (after putting <code>plot_ot_map</code> elsewhere) are useful to train solvers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I'd put them before the <code>ToyDataset</code> as not to break the flow during the reading. </p><p> </p>

Reply:
This is what I had before, Marco disagreed. I will move them above again, but I would be very happy if this is it now.